pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright 2020 Google LLC
+Copyright 2020 PerfectVIPs Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Regression script for RISC-V random instruction generator
+
+"""
+
+class riscv_instr_gen_config:
+	def __init__(self):
+		self.enable_floating_point = 1
+		self.disable_compressed_instr = 1
+		self.num_of_test  = 3
+		self.no_fence = 1
+		self.enable_sfence = 0
+		self.reserved_regs = ["ZERO","RA", "SP", "GP", "TP", "T0", "T1", "T2", "S0", "S1", "A0", "A1", "A2", "A3", "A4", "A5", "A6", "A7", \
```

Comment:
Why reserve all registers? It should be empty by default

Reply:
I took all the registers for debugging purposes.
I will change it to default.